hooks/models.js

```diff
@@ -44,7 +44,7 @@ module.exports = function Models(cb) {
         }
 
         // hand off to adapter
-        adapter.registerModel(connection.connection, modelDefinition, function(err, model) {
+        adapter.registerModel(connection.connection, model.definition, function(err, model) {
```

Comment:
i did this for the same reason as the connection above...what if 'connection' happened to be a required property for a model definition for vogels, mongoose, whatever...solely to isolate the object that the adapter is expecting when defining a model...also it seems unnecessary to me to pass the 'connection' attribute to the adapter since that property is just for the benefit of restify-microservice (unless I'm missing something)